src/main/java/windycall/Parser.java

```diff
@@ -0,0 +1,106 @@
+package windycall;
+
+/**
+ * Deals with making sense of the user command by splitting
+ * user command and translation
+ */
+public class Parser {
+
+    public Parser() {
+
+    }
+
+    /**
+     * Returns an operation type representing what users want to do
+     *
+     * @param command command input by users
+     * @return a enumeration type representing a specific operation
+     */
+    public OperationType getOperationType(String command) {
+        if (command.equals("list")) return OperationType.LIST;
+        String[] parts = command.split(" ");
+        switch (parts[0]) {
+        case "mark":
+            return OperationType.MARK;
+            //break;
+        case "unmark":
+            return OperationType.UNMARK;
+            //break;
+        case "delete":
+            return OperationType.DELETE;
+            //break;
+        case "find":
+            return OperationType.FIND;
+            //break;
+        default:
+            return OperationType.ADDTASK;
+            //break;
```

Comment:
You can consider using `Enum.valueOf()` method and `String.toUpper()` to achieve this in shorter lines of code. 😊

Reply:
Sure, I will have a try.